src/ui/game-stats-ui-handler.ts

```diff
@@ -214,201 +213,271 @@ const displayStats: DisplayStats = {
 };
 
 export default class GameStatsUiHandler extends UiHandler {
+  /** List of languages that display stats in single-column mode*/
   private gameStatsContainer: Phaser.GameObjects.Container;
   private statsContainer: Phaser.GameObjects.Container;
 
-  private statLabels: Phaser.GameObjects.Text[];
-  private statValues: Phaser.GameObjects.Text[];
+  /** The number of rows enabled per page. */
+  private static readonly ROWS_PER_PAGE = 9;
+
+  private statLabels: Phaser.GameObjects.Text[] = [];
+  private statValues: Phaser.GameObjects.Text[] = [];
 
   private arrowUp: Phaser.GameObjects.Sprite;
   private arrowDown: Phaser.GameObjects.Sprite;
 
-  constructor(mode: UiMode | null = null) {
-    super(mode);
+  /** Whether the UI is single column mode */
+  private get singleCol(): boolean {
+    const resolvedLang = i18next.resolvedLanguage ?? "en";
+    // NOTE TO TRANSLATION TEAM: Add more languages that want single-column here: add `|| resolvedLang === "lang"`
+    return resolvedLang === "ru";
```

Comment:
I'd like to do it this way instead.
```suggestion
    // NOTE TO TRANSLATION TEAM: Add more languages that want to display
    // in a single-column inside of the `[]` (e.g. `["ru", "fr"]`)
    return ["ru"].contains(resolvedLang);
```

Reply:
That's slower and does an array allocation.

Reply:
It was done this way somewhere else too (in one of your UI PRs), and game stats is a low-traffic menu (though I guess the handlers are still all loaded at once here atm unfortunately).

Given the TL team (or at least adri, who edits on behalf of the TL team) use the GitHub editor and not an IDE, I want it to be as simple as possible to edit.

Reply:
That's a fair point!